fix: Corrected console text and fixed broken link (#57)
konveyor/move2kube-website: docs/tutorials/customizing-the-output/add-custom-files-directories-in-custom-locations.md

nav_order: 4
---

> # Generating a custom Helm chart to illustrate custom files and directories could be added in custom locations

Reviewer: ```suggestion
# Generating custom Helm charts to illustrate adding custom files and directories in custom locations
```

{% endraw %}
```

> Next step [Customizing generated Dockerfile and behavior of inbuilt transformer](/tutorials/customizing-the-output/custom-dockerfile-change-inbuilt-behavior)

Reviewer: This is currently the last transformer. So it is not supposed to have the next step. We can remove the line.
```suggestion
```
